platform-configs/protectli-vp2430.robot

```diff
@@ -0,0 +1,47 @@
+*** Settings ***
+Resource    include/protectli-pro.robot
+Resource    include/protectli-common.robot
+
+
+*** Variables ***
+${FLASH_SIZE}=                      ${16*1024*1024}
+${WIFI_CARD_UBUNTU}=                Intel Corporation Wi-Fi 6 AX200 (rev 1a)
+${LTE_CARD}=                        Qualcomm, Inc. MDG200
+${DEVICE_USB_KEYBOARD}=             Logitech, Inc. Keyboard K120
+${INITIAL_CPU_FREQUENCY}=           3300
+${MAX_CPU_TEMP}=                    95
+${FLASHING_METHOD}=                 external
+
+# eMMC driver support
+${E_MMC_NAME}=                      BJTD4R
+
+@{ATTACHED_USB}=                    ${TBD}
+
+${DMIDECODE_SERIAL_NUMBER}=         N/A
+${DMIDECODE_FIRMWARE_VERSION}=      N/A
+${DMIDECODE_PRODUCT_NAME}=          VP2430
+${DMIDECODE_RELEASE_DATE}=          N/A
+${DMIDECODE_MANUFACTURER}=          Protectli
+${DMIDECODE_VENDOR}=                3mdeb
+${DMIDECODE_FAMILY}=                N/A
```

Comment:
Should be Vault Pro

Reply:
updated
https://github.com/Dasharo/open-source-firmware-validation/pull/554/commits/58b92e5bb5a08853dc688d17c9866b2ed12f3c32